libpod/container.go

```diff
@@ -252,6 +252,19 @@ type ContainerConfig struct {
 	UTSNsCtr    string `json:"utsNsCtr,omitempty"`
 	CgroupNsCtr string `json:"cgroupNsCtr,omitempty"`
 
+	// Whether container shares an NS with the pod
+	// NetNsPod conflicts with the CreateNetNS bool
+	// {namespace}NsPod conflicts with {namespace}NsCtr
+	// The pause container will be considered dependencies of the given container
+	// It must be started before the given container is started
+	IPCNsPod    bool `json:"ipcNsPod,omitempty"`
```

Comment:
I thought we discussed dropping these and just putting the ID in IPCNsCtr, MountNsCtr, etc

Reply:
Oh I thought we wanted to keep these. I like dropping it better I'll change it.